packages/mdc-theme/README.md

```diff
@@ -110,6 +110,38 @@ Property Name | Description
 `on-secondary` | A text/iconography color that is usable on top of secondary color
 `on-surface` | A text/iconography color that is usable on top of surface color
 
+#### `mdc-theme-prop` with CSS Custom Properties
+
+> **Note** The Sass map `$style` argument is intended *only* for use with color mixins.
+
+The `mdc-theme-prop` mixin also accepts a Sass map for the `$style` argument. The map must contain the following fields:
+
+Fields | Description
+--- | ---
+`varname` | The name of a CSS custom property
+`fallback` | A fallback value for the CSS custom property
+
+For example, the following Sass...
+
+```
+.foo {
+  @include mdc-theme-prop(color, (
+    varname: --foo-color,
+    fallback: red,
+  ));
+}
+```
+
+...will produce the following CSS...
+
+```
+.foo {
+  color: red;
+  color: var(--foo-color, red);
+}
+``
+
+The above output CSS will apply the fallback color for all supported browsers (including IE11) while allowing for CSS custom property use as a progressive enhancement. This argument type is intended for clients who need custom color application outside of the existing theme properties.
```

Comment:
The fallback terminology here might get confused with [Custom property fallback values](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#Custom_property_fallback_values). Can this be more specific?